tests/unit/adaptive_graph_of_thoughts/test_config.py

```
@@ -0,0 +1,150 @@
+# Tests for adaptive_graph_of_thoughts.config
+# Using pytest as the chosen test framework per repository conventions
+
+import importlib
+import json
+import textwrap
+from pathlib import Path
```

Ruff (0.11.9)

`pathlib.Path` imported but unused

Remove unused import: `pathlib.Path`

(F401)